app/models/application.rb

```diff
@@ -12,6 +12,10 @@ class Application < ActiveRecord::Base # rubocop:disable ClassLength
     where(benefits: false, application_type: 'income', application_outcome: %w[part full])
   }
 
+  scope :waiting_for_evidence, lambda {
+    includes(:evidence_check).references(:evidence_check).where.not(evidence_checks: { id: nil })
+  }
+
```

Comment:
Just had a chat with @colinbruce and we both agree to go with this **for now**, just to complete Evidence check flow, but we must refactor Application model, so I hope that this doesn't get left behind, especially since we've got another epic after this one is done!